build/Dockerfile.operator

```diff
@@ -0,0 +1,14 @@
+FROM centos:8
```

Comment:
Please use ubi8: registry.access.redhat.com/ubi8/ubi-minimal

Reply:
On second thought... This is only here as testing so as not to pollute the main Dockerfile. Since we are sharing the image, it would need to have everything that the handler needs.